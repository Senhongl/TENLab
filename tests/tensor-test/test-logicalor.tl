a = [[1,0,3],[4,0,-1]];
b = [[1,0,5],[-1,5,-1]];
print(a || b);