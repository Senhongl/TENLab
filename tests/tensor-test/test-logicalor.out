 1  0  1
 1  1  1
[ CPUIntType{2,3} ]
